### ISSUE TITLE
Shop under processing

### ISSUE THREAD

**Issue by pramathesh** [facebookincubator/facebook-for-woocommerce#310]
How long does the shop stay under processing? I'm only seeing the following message on my Shop Tab on Facebook. Also unable to tag any products on Facebook.

Your shop is being processed
Once processing is complete, your shop will be visible to people who visit your Page. You can still create and run ads promoting your products on Facebook.

**Comment by jieminz22** [comment]
Hi, 

It usually takes a few hours for shop processing.

**Comment by theabubakkar** [comment]
my shop is under process from 4 days. still waiting.....

**Comment by ediaz-inteliti** [comment]
Hi, any update with this issue? People who had experimented the problem how is your status now?

**Comment by theabubakkar** [comment]
Still its under process. Now its almost a week. All products are listed but not public.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by andrewimms** [comment]
I have the same issue. Still processing. I tried re-syncing but no effect.
![image](https://user-images.githubusercontent.com/16480201/38235615-abb3fa84-3719-11e8-8a73-f5fa784c2186.png)

**Comment by dmitridr** [comment]
Hello,

Thanks for all your reports. We managed to track down the issue, it was a bug in our review system.
We have a fix for newly created shops going live in a few hours and we plan to fix all *existing* shops that are having this problem by end of day tomorrow. Will update more when we've fixed everything. Thanks for your patience!

**Comment by dmitridr** [comment]
Hey all,

Happy to say that we flat out fixed the problem for 95% of people with this issue, so with luck all your shops should be live now.

If you're in the 5% we could not fix, try the following, first wait ~2-3 hours from the time of this post (it may still be in the review system because people are reviewing it). If it's still not working, reset the plugin using `Reconfigure > Advanced Options > Delete` then wait 48 hours, the shop should be processed correctly.

If that *still* does not work, which is very unlikely, please let us know by opening a new issue. That said, most likely everything is working for you now!

**Comment by ediaz-inteliti** [comment]
My shop it is working now. Thank you.

**Comment by theabubakkar** [comment]
Great! My shop is working now. Thank you for the fix...

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by trendyge** [comment]
I have the same problem! My shop is under process for 2 days. https://www.facebook.com/pg/Lifeisagift.ge/shop/

**Comment by somoschalacos** [comment]
Hello dmitrid, I am also experiencing the facebook shop processing problem. The shop is still processing for a couple of days.

**Comment by IrinaWestEnd** [comment]
Hello dmitrid

In the company shop on facebook few items precessing already more then month or even two. Can you check please if its the same bug that's you were talking about
![untitled-1](https://user-images.githubusercontent.com/40629361/41968286-d7181064-79fb-11e8-8c17-dbd7ae7b5367.jpg)
